.gitlab-ci.yml

```diff
@@ -286,12 +286,10 @@ dyn_CC7:
     # graphical libraries needed for rrdtool
     - yum install -y freetype fontconfig pixman libXrender unzip
 
-https-extension:
+extension:
   stage: extension
   dependencies:
     - compile
-  only:
```

Comment:
Now it is merged so it does not really matter, but why not using the `only` directive anymore ?

Reply:
The `upload` job needs to depend on the `extension` job, because the dependency resolves the artifacts (you get only the artifacts of the dependent jobs). If you would do a release the upload job would stall because the `extension` job would not be created.

Reply:
Isn't that a flaw in the `gitlab-registry.cern.ch/ci-tools/ci-web-deployer` image ?

Reply:
No, it's a gitlab feature. 
```yml
# Deploy to EOS folder
deployment:
    stage: deploy
    dependencies:
      - compile
      - extension
```
gitlab will download only the artifacts that you specify in the `dependencies:` tag. This has nothing to do with the image.

Reply:
ah ! Ok go it